src/olympia/devhub/templates/devhub/addons/submit/upload.html

```diff
@@ -48,6 +48,9 @@ <h3>{{ _('Upload Version') }}</h3>
     {% endif %}
   {% endif %}
   >
+  {% if theme_specific %}
+    <input type="hidden" name="theme_specific" id="upload-addon-theme-specific" value="1" />
```

Comment:
magic number as a string 😬  - is this defined anywhere as a constant ?

Reply:
Any truthy value would work, we convert it to a `bool` in the `upload` view

Reply:
Refactored to use a form field in 97f8197627d6175dd761e5a3ee459d2da239afd3